.github/workflows/mintest.yaml

```diff
@@ -15,20 +15,27 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.8"
           - "3.10"
           - "3.11"
           - "3.12"
+          - "3.13"
 
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
 
       - name: Set up Python
         uses: actions/setup-python@v4
+        if: ${{ matrix.python-version != '3.13' }}
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Set up Python 3.13
+        uses: actions/setup-python@v4
+        if: ${{ matrix.python-version == '3.13' }}
+        with:
+          python-version: "3.13.0-rc.1 - 3.13"
```

Comment:
why you went with this mode? to preserve the label?

Reply:
Here probably it is pure bikeshedding :slightly_smiling_face:. Bit in `tests.yaml` it is sort of important because if you set branch protection based on names, it is annoying that they change every time you go from prerelease to release.